### ISSUE TITLE
[Bug] Table: The nvda speech viewer cannot read elements on the same line of a row header

### ISSUE THREAD

**Issue by GuillaumeKESTEMAN** [AxaFrance/react-toolkit#1101]
### Issue and Steps to Reproduce

La visionneuse de parole de NVDA n'arrive pas à lire certaines lignes à côté d'un header row.

Quand ce n'est pas un header row tout fonctionne correctement.
Quand on enlève les propriétés css `position: relative` et `padding: 1rem` de la classe `af-table__th` sur le th ça refonctionne correctement pour les éléments de la même ligne.

(j'ai un doute  pour le `padding` par contre pour la `position` j'en suis quasiment sûr)

#### Versions

V2.1.0

#### Screenshots

![nvda-table](https://github.com/AxaFrance/react-toolkit/assets/81412571/b4f517fd-c7cf-417b-8ec0-81b18dc51ce9)

#### Expected

La visionneuse de parole arrive à lire tous les éléments sur la même ligne qu'un header de ligne.

#### Actual

La visionneuse de parole n'arrive pas à lire tous les éléments sur la même ligne qu'un header de ligne.

#### Additional Details

- Installed packages
